app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,23 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        let resendEmail = await this.loader.post('orders/resend-email', payload);
+        if (resendEmail.status) {
```

Comment:
@uds5501 update this to the correct error handling. If `this.loader.post` succeeds it should be  considered a success. Any error should be in the catch block

Reply:
@niranjan94 Did it!